pkg/apis/kops/componentconfig.go

```diff
@@ -163,6 +163,8 @@ type KubeProxyConfig struct {
 	HostnameOverride string `json:"hostnameOverride,omitempty" flag:"hostname-override"`
 	// Master is the address of the Kubernetes API server (overrides any value in kubeconfig)
 	Master string `json:"master,omitempty" flag:"master"`
+	// Disable allows disabling kube-proxy
+	Disable bool `json:"disable,omitempty" flag:"disable"`
```

Comment:
Shouldn't we have an enabled flag? If I remember bools default to true, so if we have an enabled flag then it default to enabled.

The other option is to have a *bool.

Reply:
Bool defaults to false.

I think the Disabled flag was probably correct.

*bool vs bool, we aren't really using the tri-state here, so bool would have been fine.   (And if nil == false we have the `fi.BoolValue` helper)